### DISCUSSION TITLE
想自己画一些元素到table上，s2有提供接口吗

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [antvis/S2#1769] [Q&A] [answered]
想自己画一些元素到table上，s2有提供接口吗，

类似下面这种蓝色的框。
![image](https://user-images.githubusercontent.com/15354455/190964232-994b3dd1-08de-4ea4-bb04-003b4a18cd4f.png)

**Comment by lijinke666** [comment, marked as answer]
https://s2.antv.vision/zh/docs/api/graphic

https://g.antv.vision/zh/examples/shape/rect#basic
